### DISCUSSION TITLE
For adding new CGM

### DISCUSSION THREAD

**Discussion by tuliper** [NightscoutFoundation/xDrip#3151] [General]
Hi, I am the mother of a teenager with type 1 diabetes living in Korea and the representative of KST1D (Korean Society of Type 1 Diabetes). I am also Somyeong's mom who participated in the xDrip+ Korean translation.  In fact, a Korean company has finally developed, received approval for, and started selling CGM (CareSens Air, from now on we will refer to it as Air). Please check the site below for product information.  https://caresensair.com/en_US/  So we want to connect Air to xDrip+.  We would like to link H/W DataSource to retrieve blood sugar from the Air server. I would like to add it similar to the way you enter the username and password and retrieve data from the Dexcom Share server.  I would be grateful if you could provide me with the email address of someone I can communicate with regarding this.

**Comment by Navid200** [comment]
You can reach me at:
xDrip dot Navid at gmail dot com

I will have a few questions.  And I can put you in contact with the right person.
But, my first question will be why your interest in using xDrip as a follower rather than a direct collector.

**Comment by jamorham** [comment]
@tuliper Do you have cooperation of the manufacturer?

**Reply by Navid200** [reply]
@jamorham I have forwarded the email to you.
